node_modules/.yarn-integrity

```diff
@@ -0,0 +1,12 @@
+{
+  "systemParams": "darwin-x64-83",
+  "modulesFolders": [],
+  "flags": [],
+  "linkedModules": [
+    "whiskeyshop"
+  ],
+  "topLevelPatterns": [],
+  "lockfileEntries": {},
+  "files": [],
+  "artifacts": {}
+}
```

Comment:
- [ ] according to the GENERAL REQUIREMENTS, the node_modules folder should be added to the .gitignore file.